.env.example

```diff
@@ -24,8 +24,8 @@ FINALIZATION_PERIOD_SECONDS=12
 ENABLE_GOVERNANCE=false
 
 # Finality gadget RPC
-# note: use external host address to avoid cross-container calls
-BBN_FINALITY_GADGET_RPC=11.22.33.44:50051
+# note: keep empty string before the Finality Gadget is up
+BBN_FINALITY_GADGET_RPC=""
```

Comment:
just `BBN_FINALITY_GADGET_RPC=` should be good enough

Reply:
i just asked Cursor, it said if we unset this env var, in the generated L2 json file, the `babylonFinalityGadgetRpc` will be the literal string "$BBN_FINALITY_GADGET_RPC", like:
```json
"babylonFinalityGadgetRpc": "$BBN_FINALITY_GADGET_RPC"
```

Reply:
i tested it, you are right, `BBN_FINALITY_GADGET_RPC=` works fine, Cursor is wrong